Makefile

```diff
@@ -346,29 +346,41 @@ build-manifest: kustomize
 	./hack/build-manifest.sh "${OPTS}"
 .PHONY: build-manifest
 
-cluster-clean: build-manifest
-	./hack/cluster-clean.sh
-.PHONY: cluster-clean
-
 cluster-deploy:
 	./hack/cluster-deploy.sh
 .PHONY: cluster-deploy
 
-cluster-sync:
-	./hack/cluster-sync.sh
```

Comment:
Why cluster-sync was removed? This is very important for development purpose to redeploy Kepler to verify the functionalities.

Reply:
to redeploy kepler without building images you can run `make cluster-deploy` with [NO_BUILD](https://github.com/sustainable-computing-io/kepler/pull/909/files#diff-be559fd8e5c86cd4bfb53d341659b318f4c5963a8da53ca413c19528590edd4bR35:~:text=declare-,NO_BUILD,-%3Dfalse) to true. This will redeploy Kepler using the generated manifests.